### ISSUE TITLE
Shorthand to access ids e fields metadata 

### ISSUE THREAD

**Issue by euduardo** [herbsjs/herbsjs.github.io#120]
**Is your feature request related to a problem? Please describe.**
It would be nice to have an API to access the Ids and Fields metadata from an entity.

You can already access those metadata, but the API is not user friendly.

For example, to check if an property is an id, it is the necessary code:
```js
const User = entity('User', {
    id: id(Number),
   name: field(String)
})

const user = new User()

const isId = user.__proto__.meta.schema.id.options.isId
```

**Describe the solution you'd like**
I would like to access those fields from a static field or method in the entity class. Something like:
```js
const User = entity('User', {
    id: id(Number),
   name: field(String)
})

const userIds = User.schema.ids
const userFields = User.schema.fields
``` 

**Additional context**
There are some open issues that could use this feature without reimplementing it again.

**Comment by dalssoft** [comment]
Is it duplicated? https://github.com/herbsjs/gotu/issues/60

Why keep both?

**Comment by dalssoft** [comment]
To keep track: PR for the feature is here https://github.com/herbsjs/gotu/pull/57

**Comment by dalssoft** [comment]
as far as I understand, this issue is related to documentation improvements of a feature already implemented by @euduardo.

@italojs please, do not close issues without a reason or checkin with others if that is ok.

since this docs are still needed, I'm opening this issue again.

**Comment by sanderiw** [comment]
I can do this. I will send the PR this week.

**Comment by jhomarolo** [comment]
@sanderiw just check with @euduardo how much it takes to finish this issue.

**Comment by sanderiw** [comment]
I talked to @euduardo and he explaneid everythig about this issue to me. Since he's almost finished this issue already, if anything comes up from the CR I can take from there.

**Comment by dalssoft** [comment]
@euduardo @sanderiw did you check my comments regarding the duplicated info?

**Comment by jhomarolo** [comment]
Released on #127
